src/docs.md

```diff
@@ -378,6 +378,15 @@ To invalidate caches and reload:
 cadenzaClient.reload({ invalidateCaches: true });
 ```
 
+### Close the Cadenza Dialog or Window That Embeds the Custom Application
+
+<small>API: [CadenzaClient#closeMe](./classes/CadenzaClient.html#closeMe)</small>
+
+Sends a message to parent Cadenza window to close the window containing this application.
```

Comment:
The Cadenza window is not necessarily the parent window. It can also be the window.opener.